src/db/tables/onerep_scans.ts

```diff
@@ -17,12 +17,13 @@ import {
 } from "knex/types/tables";
 import { RemovalStatus } from "../../app/functions/universal/scanResult.js";
 import { getQaCustomBrokers, getQaToggleRow } from "./qa_customs.ts";
+import { DataBrokerRow } from "../../knex-tables";
 
 const knex = createDbConnection();
 
 export interface LatestOnerepScanData {
   scan: OnerepScanRow | null;
-  results: OnerepScanResultRow[];
+  results: OnerepScanResultRow[] | (OnerepScanResultRow & DataBrokerRow)[];
```

Comment:
I think this is fine, but if you want to construct a very specific type (e.g. just the `data_broker_status` key and not a full union of the two types) you could use https://www.typescriptlang.org/docs/handbook/utility-types.html#picktype-keys

Reply:
Thought about this, I think it would be less readable if we use PICK (it would be a long list), same with EXCLUDE etc